src/libs/elektra/kdb.c

```diff
@@ -562,12 +565,11 @@ KDB * kdbOpen (const KeySet * contract, Key * errorKey)
  * Closes the session with the Key database.
  *
  * @pre The handle must be a valid handle as returned from kdbOpen()
- *
  * @pre errorKey must be a valid key, e.g. created with keyNew()
  *
  * This is the counterpart of kdbOpen().
  *
- * You must call this method when you finished your affairs with the key
+ * You must call this method when you are finished working with the key
```

Comment:
Sometimes Key is uppercase, sometimes lowercase.

```suggestion
 * You must call this method when you are finished working with the Key
```